bin/antigen.zsh

```diff
@@ -975,13 +975,30 @@ antigen-cleanup () {
 }
 
 antigen-help () {
+  antigen-version
+
   cat <<EOF
-Antigen is a plugin management system for zsh. It makes it easy to grab awesome
-shell scripts and utilities, put up on github. For further details and complete
-documentation, visit the project's page at 'http://antigen.sharats.me'.
 
+Antigen is a plugin management system for zsh. It makes it easy to grab awesome
+shell scripts and utilities, put up on Github.
+
+Usage: antigen <command> [args]
+
+Commands:
+  cache-gen    Generate Antigen's cache with currently loaded bundles.
```

Comment:
Is there a reason for omitting `antigen-bundle` and `antigen-apply`? I'd also add `antigen-selfupdate`.